common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -311,6 +311,6 @@ public int chunkFetchHandlerThreads() {
       conf.getInt("spark.shuffle.server.chunkFetchHandlerThreadsPercent", 100);
     return (int)Math.ceil(
      (this.serverThreads() > 0 ? this.serverThreads() : 2 * NettyRuntime.availableProcessors()) *
-     chunkFetchHandlerThreadsPercent/(double)100);
+     (double)chunkFetchHandlerThreadsPercent/100);
```

Comment:
Write 100.0 here for extra clarity, and put spaces after all the casts here. Otherwise, what's the issue with this expression?

Reply:
`lgtm.com` and I are afraind `(...) * chunkFetchHandlerThreadsPercent` may cause overflow since it is `int * int`. After this change, it will be `double * double`.

Reply:
Oh it's because it's really (threads * percent) / 100. Maybe this can be simplified by breaking out the `(this.serverThreads() ...)` into a local variable, then writing as threads * (percent / 100.0)